cypress/integration/onboarding.spec.js

```diff
@@ -4,6 +4,8 @@ context("onboarding", () => {
     passphrase: "curled",
   };
 
+  const radUrnRegex = /hwd[a-z0-9]{56}/;
```

Comment:
Yes, the name for this variable is wrong. I’ve renamed it to `radIdRegex` in https://github.com/radicle-dev/radicle-upstream/pull/1120/commits/1c3a738bb1a1a70a504b689abfc7390d19b7ec26. We can’t match the full RadURN with the `rad:git:` prefix because the UI only shows the ID.

Reply:
I think this was my doing a while ago before I knew the exact encoding for our URNs. The `rad:git:` part is already stripped out as the test is looking at the UI.

We're validating peers with this in another part of the code:
`/[1-9A-HJ-NP-Za-km-z]{54}/`

I wonder if we could adapt that here as well for the URN hash: `/[1-9A-HJ-NP-Za-km-z]{56}/`?